### PR TITLE
Add influx integration

### PR DESCRIPTION
**Pull request by dmitri86git**

### CODE REVIEW

**Comment by dmitri86git** [comment]
Connected to: 
https://github.com/YotpoLtd/yotpo-api/pull/1742
https://github.com/YotpoLtd/jenkinsdsl/pull/47
https://github.com/YotpoLtd/yapps-cookbook/pull/55

**Comment by dmitri86git** [comment]
@ronbarab 
Hey can you please review it?

File: lib/red_cross.rb [hunk 1]
```diff
@@ -3,20 +3,30 @@
...
+        Configuration.segment.identify(attrs)
+      end
+
+      def flush
+        Configuration.segment.flush
+      end
+
+      # Measurement has to include event and properties
+      def monitor(attrs)
+        data = {values: {count: 1}, tags: attrs[:properties]}
```
Comment (RonBarabash) [inline lib/red_cross.rb]: @dmitri86git this value always needs to be 1?

Comment (dmitri86git) [inline lib/red_cross.rb]: I can set it 1 as a default and add option to use attrs[:count]

Comment (RonBarabash) [inline lib/red_cross.rb]: @dmitri86git Lets do that.

**Comment by RonBarabash** [comment]
⭐

**Pull request merged**